docs/book/pattern/callback-cache.md

```diff
@@ -5,42 +5,40 @@ The callback cache pattern caches the results of arbitrary PHP callables.
 ## Quick Start
 
 ```php
-use Laminas\Cache\PatternFactory;
+use Laminas\Cache\Pattern\CallbackCache;
 use Laminas\Cache\Pattern\PatternOptions;
+use Laminas\Cache\Storage\StorageInterface;
 
-// Via the factory:
-$callbackCache = PatternFactory::factory('callback', [
-    'storage'      => 'apc',
-    'cache_output' => true,
-]);
+/** @var StorageInterface $storage */
+$storage = null; // Can be any instance of StorageInterface
```

Comment:
The pattern section contains an introduction page, it would be good if a standard storage adapter for the documentation of the cache patterns is defined there. This can be done in blockquote and can then be linked.

For laminas-i18n I used the filesystem adapter: https://docs.laminas.dev/laminas-i18n/translator/caching/

Reply:
Hm, I've tried something but not sure if thats what you had in mind. Feel free to review.